spec/acceptance/ini_subsetting_spec.rb

```diff
@@ -120,7 +120,7 @@
     MANIFEST
 
     it 'applies the manifest twice' do
-      run_shell("echo -e [one]\\\\nkey = alphabet betatrons > #{basedir}/ini_subsetting.ini", expect_failures: true)
+      run_shell("printf \"[one]\\nkey = alphabet betatrons\n\" > #{basedir}/ini_subsetting.ini", expect_failures: true)
```

Comment:
Quotes are not easy to read but this look inconsistent: the argument passed to printf(1) is `"[one]\\nkey = alphabet betatrons\n"`: the first `\\n` is passed to printf(1) as `\n` and the second one `\n` is passed as a new line.

For consistency, I would suggest using `\\n` in both occurrences:

```suggestion
      run_shell("printf \"[one]\\nkey = alphabet betatrons\\n\" > #{basedir}/ini_subsetting.ini", expect_failures: true)
```

That way we pass a single line string with two `\n` and printf will turn them into new lines.

Reply:
Except this minor consistency issue, +1 for a portable way to output the document!